### PR TITLE
Refactor Markdownly to use Redux

### PR DESCRIPTION
**Pull request by jwmcconnell**

### CODE REVIEW

**Review by dannyhogan — APPROVED** [review]
Seems good!

File: src/containers/AllDocuments.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { connect } from 'react-redux';
```
Comment (mehtaphysical) [inline src/containers/AllDocuments.js]: Why is this called all documents? It is a single document connected to redux

Comment (jwmcconnell) [reply src/containers/AllDocuments.js]: I assumed that it made sense in the context that we were going to be using multiple markdown documents in the future.  I can change it though if it doesn't work.

**Review by mehtaphysical — APPROVED** [review]

File: src/containers/AllDocuments.js (hunk 1 above)
Comment (mehtaphysical) [reply src/containers/AllDocuments.js]: Maybe it will, but don't try to predict the future. Just accomplish the task at hand

**Pull request merged**